Vagrantfile

```diff
@@ -49,6 +49,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     "os"                => "fedora",
     "dev_cluster"       => false,
     "dind_dev_cluster"  => ENV['OPENSHIFT_DIND_DEV_CLUSTER'] || false,
+    "network_plugin"    => ENV['OPENSHIFT_NETWORK_PLUGIN'] || ENV['OPENSHIFT_SDN'] || "",
```

Comment:
if we're getting rid of differences between dind and non-dind, you should unify this as well